--- /opt/victronenergy/gui/qml/PageMain._qml
+++ /opt/victronenergy/gui/qml/PageMain.qml
@@ -57,6 +57,11 @@
 	Component {
 		id: vebusPage
 		PageVebus {}
+	}
+
+	Component {
+		id: vebusPageHm
+		PageVebusHm {}
 	}
 
 	Component {
@@ -146,5 +151,5 @@
 		{
 		case DBusService.DBUS_SERVICE_MULTI:
-			page = vebusPage
+			page = service.name.indexOf("vebus.MicroPlus") > 0?vebusPageHm:vebusPage
 			break;
 		case DBusService.DBUS_SERVICE_MULTI_RS:
